README.md

```diff
@@ -11,35 +11,18 @@ is an XML-based open-standard for transferring identity data between two parties
 an identity provider (IdP) and a service provider (SP).
 
 This charm simplifies configuration of SAML SPs by providing a single point
-of configuration for all the requirers using the same SAML entity. It can be
-deployed on many different Kubernetes platforms, from [MicroK8s](https://microk8s.io)
-to [Charmed Kubernetes](https://ubuntu.com/kubernetes), public cloud Kubernetes
-offerings and virtual machines or bare metal.
+of configuration for all the requirers using the same SAML entity.
 
-As such, the charm makes it easy to manage and propagate SAML configuration, while
-giving the freedom to deploy on the substrate of their choice.
+## Get started
 
-For DevOps or SRE teams this charm will make operating any charm leveraging SAML
-authentication simple and straightforward through Juju's clean interface.
+You can follow the tutorial [here](https://charmhub.io/saml-integrator/docs/tutorial-getting-started).
 
```

Comment:
Are there any basic operations (e.g. actions) or integrations worth mentioning for this charm?

Reply:
No actions. And no integrations as a requirer